archive/src/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
Thanks for initializing the index.js for the fresh repo, will help with initial structuring for new core app